apps/admin-gui/src/app/vos/pages/vo-select-page/vo-select-page.component.ts

```diff
@@ -38,4 +47,17 @@ export class VoSelectPageComponent implements OnInit {
   applyFilter(filterValue: string) {
     this.filterValue = filterValue;
   }
+
+  onCreateVo() {
+    const dialogRef = this.dialog.open(CreateVoDialogComponent, {
+      width: '1000px',
```

Comment:
I think this is too much. 600px would be enough.
```suggestion
      width: '600px',
```